bot/bastion/entrypoint.sh

```diff
@@ -7,13 +24,19 @@ export INTERNAL_IP
 
 # Replace Startup Variables
 MODIFIED_STARTUP=$(echo -e $(echo -e ${STARTUP} | sed -e 's/{{/${/g' -e 's/}}/}/g'))
-echo -e ":/home/container$ ${MODIFIED_STARTUP}"
+echo -e "${YELLOW}:/home/container${NC} ${MODIFIED_STARTUP}"
 
 # start mongo
-mongod --fork --dbpath /home/container/mongodb/ --port 27017 --logpath /home/container/mongod.log && until nc -z -v -w5 127.0.0.1 27017; do echo 'Waiting for mongodb connection...'; sleep 5; done
+echo -e "${BLUE}-------------------------------------------------${NC}"
+echo -e "${YELLOW}starting MongoDB...${NC}"
+echo -e "${BLUE}-------------------------------------------------${NC}"
+mongod --fork --dbpath /home/container/mongodb/ --port 27017 --logpath /home/container/mongod.log --logRotate reopen --logappend && until nc -z -v -w5 127.0.0.1 27017; do echo 'Waiting for mongodb connection...'; sleep 5; done
 
 # Run the Server
+echo -e "${BLUE}-------------------------------------------------${NC}"
+echo -e "${YELLOW}BastionBot starting...${NC}"
+echo -e "${BLUE}-------------------------------------------------${NC}"
 eval ${MODIFIED_STARTUP}
 
 # stop mongo
-mongo --eval \"db.getSiblingDB('admin').shutdownServer()\
+mongo --eval "db.getSiblingDB('admin').shutdownServer()"
```

Comment:
pls change this to

`mongod --shutdown`

Reply:
does it work if mongo is running in the background?

Reply:
> does it work if mongo is running in the background?

i had an error, that "mongo" was not found; after changing it to mongod --shutdown it worked

But test it. Mongo 6 need mongod --shutdown